cpp/neighbors_module/algorithm/neighbors.cpp

```diff
@@ -1 +1,91 @@
 #include "neighbors.hpp"
+
+#include <fmt/format.h>
+#include <unordered_set>
+
+bool Known(const mgp::Node &node, std::vector<std::unordered_set<mgp::Node>> &list) {
+  for (auto element : list) {
+    if (element.contains(node)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+void DetermineDirection(mgp::List &rel_types, std::unordered_set<std::string_view> &in_rels,
+                        std::unordered_set<std::string_view> &out_rels) {
+  if (rel_types.Empty()) {
+    rel_types.AppendExtend(mgp::Value(""));
+  }
```

Comment:
Why is this necessary? Just asking because later on it ends up in `in_rels` and `out_rels` and there are several occasions where you check if those two are empty